pkg/storage/bloom/v1/index_querier.go

```diff
@@ -52,8 +52,6 @@ func (it *LazySeriesIter) Seek(fp model.Fingerprint) error {
 		return header.ThroughFp >= fp
 	})
 
-	page := it.b.index.pageHeaders[desiredPage]
-
 	switch {
 	case desiredPage == len(it.b.index.pageHeaders):
```

Comment:
Could it be possible that `desiredPage` is greater that `len(it.b.index.pageHeaders)`?

Reply:
No, according to the [search](https://pkg.go.dev/sort#Search) fn used.